lib/display.ex

```diff
@@ -21,9 +21,24 @@ defmodule Display do
   defp name(module), do: name(Atom.to_string(module))
 
   def show_failure(failure, module, name) do
+    intro(module, Tracker.visited)
```

Comment:
If it works as it is maybe don't even worry about it. I suppose you could add a note or a comment or something that suggests a refactor in the future if you wanted to.  It's probably not a big deal